dts/bindings/watchdog/nordic,npm2100-wdt.yaml

```
@@ -0,0 +1,6 @@
+# Copyright (c) 2024 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+description: nPM2100 Watchdog
+
+compatible: "nordic,npm2100-wdt"
```

AFAIK `base.yaml` should be included for all drivers.

```suggestion
compatible: "nordic,npm2100-wdt"

include: base.yaml
```

---

I guess only if they use any base.yaml property

---

Yeah, it uses at least the `reg` property from `base.yaml`.

---

It doesn't use any properties other than what is inherited from the mfd parent device. I don't know if that involves base.yaml.

---

You are right. Should be ok not to import base.yaml.